configure.ac

```diff
@@ -1276,9 +1279,15 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <openssl/ssl.h>]],
                     AC_MSG_RESULT([no])
                   ])
 
+AC_ARG_ENABLE([quic],
+  AS_HELP_STRING([--disable-quic], [disable QUIC support]), [
```

Comment:
I'm fine with having this option if you want to completely disable QUIC, but it doesn't make much sense to me because I think there's only overhead on startup if ports for QUIC are not configured.

Can you tell me what made you want to add this option?

Reply:
Purely for building in Xcode. I have a project I maintain and wanted to continue to use Xcode for developing the BoringSSL changes, but using that would force QUIC support. And I'm lazy and didn't want to add the QUIC files to it (especially since default openssl doesn't have the appropriate support).

Reply:
>  didn't want to add the QUIC files to it

Can you tell me a bit more detail about this part? Do those files bother Xcode somehow?

Reply:
It's a manual process of add/removing files. The files themselves are fine as long as they're not part of the build, which I need if I'm building against OpenSSL). but that requires either me maintaining a separate branch/copy of the project for QUIC support. 

If this is controversial compile time option, I'll just drop it from the changeset before I merge.

Reply:
I'm purely curious why the option is needed, and I still don't get the problem... maybe because I don't use Xcode, but if it's helpful for some situations I don't against adding it as long as you are not making changes that are incompatible with QUIC support (having QUIC support as todo is fine).

I'd make another PR for this compile time option so that people can easily cherrypick the change for their branch.

Current expected behavior of the configure script is to enable QUIC support whenever SSL library supports QUIC. In other words, the files should not be part of the build if the library doesn't support QUIC even if you use BoringSSL.

Reply:
I've removed the option from this PR (and will work with a local copy of if it in the future)